docs/libFuzzerIntegration.rst

```diff
@@ -0,0 +1,50 @@
+libFuzzer Integration
+---------------------
+
+Swift compiler comes with builtin ``libFuzzer`` integration,
+using a manually constructed Objective-C bridge.
+
+Suppose we would like to fuzz-test a Swift function called ``fuzzMe`` from the
+file ``myfile.swift``.
+First, we decorate the function with ``@objc`` and make sure it accepts
+``UnsafePointer<CChar>`` and ``CInt`` parameters:
+
+.. code-block:: swift
+
+    @objc(MyClass) class MyClass {
+      static func fuzzMe(Data: UnsafePointer<CChar>, Size: CInt) {
+        // Test our code using provided Data.
+      }
+    }
+
+Next, we call ``[MyClass fuzzMeWithData:Size:]`` (note the function name
+includes an auto-generated postfix; use the ``-emit-objc-header`` flag
+to generate the interface automatically) from an entry point function inside
+a separate Objective-C file ``fuzzwrapper.m``:
+
+.. code-block:: objective-c
+
+    @interface MyClass
+    + (void) fuzzMeWithData:(char*)Data Size:(int)Size;
+    @end
+    int LLVMFuzzerTestOneInput(const char *Data, long Size) {
+      [MyClass fuzzMeWithData:Data Size:Size];
+    }
+
```

Comment:
Oh, gosh, I forgot `@_cdecl` is actually hooked up in-tree. Even if we haven't ironed out all the bugs it should be fine for UnsafePointer and CInt (CUnsignedLong?). Sorry, George!